website/content/installation.md

```diff
@@ -107,7 +107,7 @@ Example:
 Schema: see [`portForwardings`](
 https://github.com/rtr7/router7/blob/f86e20be5305fc0e7e77421e0f2abde98a84f2a7/internal/netconfig/netconfig.go#L431)
 
-Please be aware that Hairpinning is currently not supported (see issue [#53 Support for Hairpinning](https://github.com/rtr7/router7/issues/53]))
+Please be aware that Hairpinning is currently not supported (see issue [#53 Support for Hairpinning](https://github.com/rtr7/router7/issues/53))
```

Comment:
That sentence can actually be removed entirely, as hairpinning is now supported.

Reply:
I removed it, let me know if there is anything else I could do